<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DipoleFitControl</class>
 <widget class="QWidget" name="DipoleFitControl">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>240</width>
    <height>719</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>200</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>254</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dipole Fit Control</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Select Dipole Fit</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QComboBox" name="m_qComboBoxFitSelector"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="m_qGroupBoxProperties">
     <property name="title">
      <string>Properties</string>
     </property>
     <layout class="QFormLayout" name="formLayout_2">
      <item row="1" column="0">
       <widget class="QLabel" name="m_qLabelMeasurementPath">
        <property name="text">
         <string>Measurement:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="2">
       <widget class="QLineEdit" name="m_qLineEditSTCFile">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QPushButton" name="m_qPushButtonBrowseMeasurementPath">
        <property name="text">
         <string>Browse</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="m_qLabelUseRaw">
        <property name="text">
         <string>Use raw:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QCheckBox" name="m_qCheckBoxUseRaw">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="m_qLabeSetNumber">
        <property name="text">
         <string>Set number:</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QSpinBox" name="m_qSpinBoxSetNumber"/>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="m_qLabeIncludeMeg">
        <property name="text">
         <string>include MEG:</string>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QCheckBox" name="m_qCheckBoxIncludeMeg">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="7" column="0">
       <widget class="QLabel" name="m_qLabeIncludeEeg">
        <property name="text">
         <string>include EEG:</string>
        </property>
       </widget>
      </item>
      <item row="7" column="1">
       <widget class="QCheckBox" name="m_qCheckBoxIncludeEeg">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="8" column="0">
       <widget class="QLabel" name="m_qLabeTMax">
        <property name="text">
         <string>t max:</string>
        </property>
       </widget>
      </item>
      <item row="8" column="1">
       <widget class="QDoubleSpinBox" name="m_qDoubleSpinBoxTMax">
        <property name="decimals">
         <number>6</number>
        </property>
        <property name="minimum">
         <double>-99.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="9" column="0">
       <widget class="QLabel" name="m_qLabeTMin">
        <property name="text">
         <string>t min:</string>
        </property>
       </widget>
      </item>
      <item row="9" column="1">
       <widget class="QDoubleSpinBox" name="m_qDoubleSpinBoxTMin">
        <property name="decimals">
         <number>6</number>
        </property>
        <property name="minimum">
         <double>-99.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="10" column="0">
       <widget class="QLabel" name="m_qLabeBMax">
        <property name="text">
         <string>b max:</string>
        </property>
       </widget>
      </item>
      <item row="10" column="1">
       <widget class="QDoubleSpinBox" name="m_qDoubleSpinBoxBMax">
        <property name="decimals">
         <number>6</number>
        </property>
        <property name="minimum">
         <double>-99.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="12" column="0">
       <widget class="QLabel" name="m_qLabeBMin">
        <property name="text">
         <string>b min:</string>
        </property>
       </widget>
      </item>
      <item row="12" column="1">
       <widget class="QDoubleSpinBox" name="m_qDoubleSpinBoxBMin">
        <property name="decimals">
         <number>6</number>
        </property>
        <property name="minimum">
         <double>-99.989999999999995</double>
        </property>
       </widget>
      </item>
      <item row="13" column="0">
       <widget class="QLabel" name="m_qLabelFitName">
        <property name="text">
         <string>Fit Name:</string>
        </property>
       </widget>
      </item>
      <item row="14" column="0" colspan="2">
       <widget class="QLineEdit" name="m_qLineEditFitName"/>
      </item>
      <item row="15" column="0" colspan="2">
       <widget class="QPushButton" name="m_qPushButtonFit">
        <property name="text">
         <string>Fit</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="fileManagementGroupBox">
     <property name="title">
      <string>File Management</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QPushButton" name="loadDipoleButton">
        <property name="text">
         <string>Load Dipole Fit</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="saveDipoleFitButton">
        <property name="text">
         <string>Save Dipole Fit</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="m_qGroupBoxBrowser">
     <property name="title">
      <string>Browser</string>
     </property>
     <layout class="QVBoxLayout" name="m_qVerticalLayoutBrowser_2">
      <item>
       <widget class="QSlider" name="m_qSliderTimePoint">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
